docs/gallery.py

```diff
@@ -378,6 +378,65 @@ def dispersion():
     savefig(oz, "dispersion")
 
 
+def postag():
+    tagged_stanzas = [
+        [
+            [
+                ('Whose', 'JJ'),('woods', 'NNS'),('these', 'DT'),
+                ('are', 'VBP'),('I', 'PRP'),('think', 'VBP'),('I', 'PRP'),
+                ('know', 'VBP'),('.', '.')
+                ],
+            [
+                ('His', 'PRP$'),('house', 'NN'),('is', 'VBZ'),('in', 'IN'),
+                ('the', 'DT'),('village', 'NN'),('though', 'IN'),(';', ':'),
+                ('He', 'PRP'),('will', 'MD'),('not', 'RB'),('see', 'VB'),
+                ('me', 'PRP'),('stopping', 'VBG'), ('here', 'RB'),('To', 'TO'),
+                ('watch', 'VB'),('his', 'PRP$'),('woods', 'NNS'),('fill', 'VB'),
+                ('up', 'RP'),('with', 'IN'),('snow', 'NNS'),('.', '.')
+                ]
+            ],
+        [
+            [
+                ('My', 'PRP$'),('little', 'JJ'),('horse', 'NN'),('must', 'MD'),
+                ('think', 'VB'),('it', 'PRP'),('queer', 'JJR'),('To', 'TO'),
+                ('stop', 'VB'),('without', 'IN'),('a', 'DT'),('farmhouse', 'NN'),
+                ('near', 'IN'),('Between', 'NNP'),('the', 'DT'),('woods', 'NNS'),
+                ('and', 'CC'),('frozen', 'JJ'),('lake', 'VB'),('The', 'DT'),
+                ('darkest', 'JJS'),('evening', 'NN'),('of', 'IN'),('the', 'DT'),
+                ('year', 'NN'),('.', '.')
+                ]
+            ],
+        [
+            [  
+                ('He', 'PRP'),('gives', 'VBZ'),('his', 'PRP$'),('harness', 'NN'),
+                ('bells', 'VBZ'),('a', 'DT'),('shake', 'NN'),('To', 'TO'),
+                ('ask', 'VB'),('if', 'IN'),('there', 'EX'),('is', 'VBZ'),
+                ('some', 'DT'),('mistake', 'NN'),('.', '.')
+                ],
+            [
+                ('The', 'DT'),('only', 'JJ'),('other', 'JJ'),('sound', 'NN'),
+                ('’', 'NNP'),('s', 'VBZ'),('the', 'DT'),('sweep', 'NN'), 
+                ('Of', 'IN'),('easy', 'JJ'),('wind', 'NN'),('and', 'CC'),
+                ('downy', 'JJ'),('flake', 'NN'),('.', '.')
+                ]
+            ],
+        [
+            [
+                ('The', 'DT'),('woods', 'NNS'),('are', 'VBP'),('lovely', 'RB'),
+                (',', ','),('dark', 'JJ'),('and', 'CC'),('deep', 'JJ'),(',', ','),
+                ('But', 'CC'),('I', 'PRP'),('have', 'VBP'),('promises', 'NNS'),
+                ('to', 'TO'),('keep', 'VB'),(',', ','),('And', 'CC'),('miles', 'NNS'),
+                ('to', 'TO'),('go', 'VB'),('before', 'IN'),('I', 'PRP'),
+                ('sleep', 'VBP'),(',', ','),('And', 'CC'),('miles', 'NNS'),
+                ('to', 'TO'),('go', 'VB'),('before', 'IN'),('I', 'PRP'),
+                ('sleep', 'VBP'),('.', '.')
+                ]
+        ]
+    ]
+    oz = PosTagVisualizer()
+    oz.fit(tagged_stanzas)
```

Comment:
Tight layout definitely does something, but maybe not what we want it to do :/  I wonder if this is related to #613?

without tight layout
![postag](https://user-images.githubusercontent.com/8760385/53245999-641dd780-367d-11e9-8249-1daab6c75bc5.png)

with tight layout
![postag](https://user-images.githubusercontent.com/8760385/53246015-726bf380-367d-11e9-8a81-9340d67357da.png)

Reply:
@rebeccabilbro as you might recall I reopened #613 because after mpl updates I was still having problems with ConfusionMatrix labels being cutoff.  I think I might have found a partial solution.  I found that adding the keyword `bbox_inches='tight'` to poof fixed the cutoff.  Maybe it will work in this case, give it a try.

```
iris_cm = ConfusionMatrix(
    model, classes=classes,
    label_encoder={0: 'setosa', 1: 'versicolor', 2: 'virginica'}
)

iris_cm.fit(X_train, y_train)
iris_cm.score(X_test, y_test)

i=iris_cm.poof(bbox_inches='tight')
```

Before `bbox_inches='tight'`
![image](https://user-images.githubusercontent.com/1000117/53255137-05635880-3693-11e9-9766-48fb69c5068e.png)


After `bbox_inches='tight'`
![image](https://user-images.githubusercontent.com/1000117/53255057-d4832380-3692-11e9-99d7-ab22fa573dcf.png)

Reply:
@lwgray thanks for the update -- and I'm glad this is a solution to the confusion matrix as well. My suggestion is to create another PR (after this is merged) to address #613 using this solution in both `ConfusionMatrix` as well as `PosTagVisualizer` - would you be interested in doing that?

Reply:
This seems to work &mdash; thanks @lwgray! Just a heads up that in `gallery.py`, the call to `poof` happens in a specialized function meant to work for all the visualizers. As a workaround, we could add optional kwargs to that function `def savefig(viz, name, gallery=GALLERY, **kwargs):` so that we can do:

```python
    oz = PosTagVisualizer(ax=newfig())
    oz.fit(tagged_stanzas)
    savefig(oz, "postag", bbox_inches="tight")
```

Reply:
thanks @rebeccabilbro for pointing this out and writing up an example...I had pointed this out to @bbengfort earlier.